subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/verification/DependencyVerificationWritingIntegTest.groovy

```diff
@@ -596,6 +596,7 @@ class DependencyVerificationWritingIntegTest extends AbstractDependencyVerificat
 <verification-metadata>
    <configuration>
       <verify-metadata>true</verify-metadata>
+      <verify-signatures>false</verify-signatures>
```

Comment:
So this means we are explicit when generating the file?
I like it indeed.

Reply:
This is indeed accidental.